app/assets/stylesheets/stories.scss

```diff
@@ -40,6 +40,26 @@
   overflow: auto;
 }
 
+.story-description em {
```

Comment:
maybe we can nest the `em`, `strong` and `ul` style the same way the `p` is nested inside `.story-description` in line 13, to remove some repetition